### DISCUSSION TITLE
glsp-examples - minimal-theia - typo in README ?

### DISCUSSION THREAD

**Discussion by rsoika** [eclipse-glsp/glsp#537] [Q&A] [answered]
Is there a typo in the README for the minimal example?

The following command did not work:

     cd glsp-client && yarn && cd minimal-theia && yarn copy:server && cd ../..

I am not the yarn expert but I think that the instruction `yarn copy:server` can not work because the package.json file did not provide such a script.

**Comment by tortmayr** [comment, marked as answer]
Hi, yes you are correct there is a currently a typo (resp. no longer existing command) in the example readme.
There is a pending PR that fixes this. (https://github.com/eclipse-glsp/glsp-examples/pull/96)

**Reply by rsoika** [reply]
ok thanks,
I see that there are a lot of additional small changes in the minimal example. Maybe this is the reason why I got no success.

For example after wiping out everything and starting from scratch with an empty folder and just copy the minimal example, I succeeded to start theia (which was a great moment for me). But the next moment when I try to build it again using yarn the theia no longer starts and I see the following messages in my browser debug console:

```
index.js:59 Failed to start the frontend application.
(anonymous) @ index.js:59
Promise.catch (async)
(anonymous) @ index.js:58
Promise.then (async)
./src-gen/frontend/index.js @ index.js:24
__webpack_require__ @ bootstrap:19
(anonymous) @ startup:4
(anonymous) @ startup:4
index.js:61 Error: Cannot apply @injectable decorator multiple times.
    at injectable.ts:8
    at DecorateConstructor (Reflect.js:541)
    at Reflect.decorate (Reflect.js:130)
    at _decorate (decorator_utils.ts:67)
    at Object.decorate (decorator_utils.ts:89)
    at Object.../node_modules/sprotty-theia/node_modules/@theia/core/lib/browser/widgets/widget.js (widget.ts:27)
    at __webpack_require__ (bootstrap:19)
    at Object.../node_modules/sprotty-theia/node_modules/@theia/core/lib/browser/widgets/index.js (index.ts:17)
    at __webpack_require__ (bootstrap:19)
    at Object.../node_modules/sprotty-theia/node_modules/@theia/core/lib/browser/dialogs.js (dialogs.ts:20)
(anonymous) @ index.js:61
Promise.catch (async)
(anonymous) @ index.js:58
Promise.then (async)
./src-gen/frontend/index.js @ index.js:24
__webpack_require__ @ bootstrap:19
(anonymous) @ startup:4
(anonymous) @ startup:4
```

**Reply by rsoika** [reply]
Ok just recognized with your new version of the `minimal-glsp-server-contribution.ts` I got rid of that problem. Thanks!
